src/python/pants/notes/2.16.x.md

```diff
@@ -46,7 +46,7 @@ available on various target types.
 A new `experimental_test_shell_command` target type has been added to allow defining arbitrary shell commands
 as tests which can be invoked using the `./pants test` goal.
 
-`shell_command` now supports the `environment` field for running in non-local environments.
+`experimental_shell_command` now supports the `environment` field for running in non-local environments.
```

Comment:
Heh, this section aims for the stable 2.16.0 release, right? so here it was correct to rename 😬

Reply:
Yep, looks like it's after dev6, so I'm going to make that change in the hope that dev7 doesn't go out today :)